packages/embeds/js/src/components/Bot.tsx

```diff
@@ -150,8 +150,24 @@ export const Bot = (props: BotProps & { class?: string }) => {
       const initialChatInStorage = getInitialChatReplyFromStorage(
         data.typebot.id
       )
-      if (initialChatInStorage) {
-        setInitialChatReply(initialChatInStorage)
+      if (
+        initialChatInStorage &&
+        initialChatInStorage.sessionUpdatedAt &&
```

Comment:
I guess we don't need to use a `sessionUpdatedAt` field. just comparing the `typebot.updatedAt` fields should be enough (API response vs in storage), if they are different then we restart the chat

Reply:
Yeah right, that would make more sense 💯 

Actually, I started to implement the same logic initially in the backend part but then realized the importance of accessing the `initialChatInStorage` object and hence moved the same logic to the frontend, that's why I compared it with the `sessionUpdatedAt` field.

Implemented the changes!